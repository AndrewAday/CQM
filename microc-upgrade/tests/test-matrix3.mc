int main(){
  fmatrix fm;
  fmatrix fm2;
  // print_mat([1.0, 2.0, 3.0; 2.0, 3.0, 4.0]);
  fm = [[1.0, 2.0, 3.0], [2.0, 3.0, 4.0]];
  fm2 = [[3.0, 5.0, 6.0], [8.0, 2.3, 5.0], [1.2, 1.3, 1.4]];
  print_mat(fm);
  print_mat(fm2);
  print_mat(fm + 2.0);
  print_mat(fm .. fm2);
  // print_mat(fm);
  return 0;
}
