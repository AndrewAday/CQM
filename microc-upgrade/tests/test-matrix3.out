1 2 3
2 3 4
  3   5   6
  8 2.3   5
1.2 1.3 1.4
3 4 5
4 5 6
22.6 13.5 20.2
34.8 22.1 32.6
